test/fixtures/output/connect/offset/update-offset-pending.golden

```diff
@@ -0,0 +1,2 @@
+Operation is PENDING. Please run `confluent connect offset status describe` command to get the latest status of the update request.
+
```

Comment:
Any idea why we have a newline here?

Reply:
Hmm, looks like this file isn't being used. Please delete it!

Reply:
Done